www/src/routes/register/Registration.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC, useState } from 'react';
```

Comment:
What's the difference between this file and /students/Registration.tsx in terms of usage?

Reply:
This was used during development before @AnushkaKhare-Eng merged from master. This file should be deleted before merging